jest.config.js

```diff
@@ -14,5 +14,8 @@ module.exports = {
     "<rootDir>/src/clean-ast.js",
     "<rootDir>/src/deprecated.js"
   ],
+  moduleNameMapper: {
+    "graceful-fs": "<rootDir>/tests_config/fs.js"
```

Comment:
If this works, maybe add a comment explaining why we do this?

Reply:
Yeah, I want to test it out in CI, and maybe do the same thing with the Rollup builds.

Reply:
Actually not in the rollup build - see https://github.com/facebook/jest/issues/2179#issuecomment-355231418

Reply:
This fixes the dev build, but the production build is still broken (because graceful-fs will be bundled in).  Should I remap `graceful-fs` for the Rollup build too?

Reply:
Maybe we could put it in `third-party.js`?

Reply:
I don't think so because we're not requiring it, [`load-json-file` is](https://github.com/sindresorhus/load-json-file/blob/master/index.js#L3)